clang_3.8/Dockerfile

```
@@ -79,8 +71,8 @@ RUN dpkg --add-architecture i386 \
     && update-alternatives --install /usr/bin/pyenv pyenv /opt/pyenv/bin/pyenv 100 \
     && PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install 3.7.5 \
     && pyenv global 3.7.5 \
-    && pip install -q --upgrade --no-cache-dir pip \
-    && pip install -q --no-cache-dir conan conan-package-tools \
+    && pip install -q --upgrade --no-cache-dir pip scikit-build \
+    && pip install -q --no-cache-dir conan conan-package-tools cmake==3.16.* \
```

```suggestion
    && pip install -q --no-cache-dir conan conan-package-tools cmake==3.16.4 \
```

You need to use a very specific version, because we use a script to update all images. Also, it keeps a more controlled scenario.